tests/test_library_libraries.py

```diff
@@ -342,7 +343,7 @@ def test_rename_changed(self):
 
     def test_tag_values(self):
         self.library.add(self.Frange(30))
-        del(self.added[:])
+        del (self.added[:])
```

Comment:
I think these braces should be removed. (`del` is a statement, not a function)

Reply:
Done (and a few more)